src/components/MapView/index.tsx

```diff
@@ -246,7 +263,9 @@ function MapView({ classes }: MapViewProps) {
       );
     }
     // let users know if their current date doesn't exist in possible dates
-    if (selectedDate && urlParams.date !== selectedDate) {
+    const urlDate = urlParams.get('date');
```

Comment:
@Ry-DS using `moment(undefined)` returns current timestamp. This might make issues and would need to redefine logic, since there is a check to include the date value in the url if it is not defined. I've included previous comment in commit a9c4a64f84eea8295bd99f6a40dfad80aef22bf4